### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nuhkoca**
* Fixed ToC names
* Added `Kotlin DSL` example of configuring test task

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
-  - [Reports](#configuring-reports)
-  - [Reports collecting](#configuring-reports-collecting)
-  - [Entire plugin](#configuring-entire-plugin)
-- [Verification](#verification)
-- [Tasks](#tasks)
+  - [Configuring JVM test task](#configuring-jvm-test-task)
+  - [Configuring reports](#configuring-reports)
+  - [Configuring reports collecting](#configuring-reports-collecting)
+  - [Configuring entire plugin](#configuring-entire-plugin)
+    - [Verification](#verification)
```
Comment (shanshin) [inline README.md:20]: Verification and the list of tasks do not relate to configuring the general behavior of the plugin (settings for all modules), so it is better to leave them at the root level.

**Comment by nuhkoca** [comment]
Done

File: README.md (hunk 1 above)
Comment (nuhkoca) [reply README.md]: Done

**Review by shanshin — APPROVED** [review]
Thanks!

**Pull request merged**